client/landing/stepper/declarative-flow/internals/hooks/use-preload-steps/index.ts

```diff
@@ -0,0 +1,79 @@
+import { SiteDetails } from '@automattic/data-stores';
+import debugFactory from 'debug';
+import { useEffect } from 'react';
```

Comment:
Should we
```suggestion
import { useEffect } from '@wordpress/element';
```
?

Reply:
I don't think it makes a difference in Calypso. Especially now that we don't work on GB.

Reply:
I would say it doesn't make a difference, especially as the recommendation now in Gutenberg is to use in `react` but we shouldn't use the excuse of the pause in contribution to stop using the Core first approach to development.